app/src/main/java/woowacourse/paint/model/brush/shape/CircleBrush.kt

```diff
@@ -0,0 +1,21 @@
+package woowacourse.paint.model.brush.shape
+
+import android.graphics.Paint
+import android.graphics.Path
+import android.view.MotionEvent
+
+class CircleBrush(override var paint: Paint) : ShapeBrush {
+
+    override var path: Path = Path()
+    override var startX: Float = DEFAULT_COORDINATE
+    override var startY: Float = DEFAULT_COORDINATE
+
+    override fun moveDrawing(event: MotionEvent) {
+        path.reset()
+        path.addCircle(startX, startY, event.x - startX, Path.Direction.CCW)
```

Comment:
```suggestion
        path.addCircle(startX, startY, abs(event.x - startX), Path.Direction.CCW)
```
위와 같이 바꿔주면 어떤 좌표로 드래그 해도 원이 그려집니다~